diff --git a/rtl/include/quadrilatero_instr_pkg.sv b/rtl/include/quadrilatero_instr_pkg.sv
index 3216407..2e288d9 100644
--- a/rtl/include/quadrilatero_instr_pkg.sv
+++ b/rtl/include/quadrilatero_instr_pkg.sv
@@ -48,7 +48,7 @@ package quadrilatero_instr_pkg;
   // localparam [31:0] MLD_B              = 32'b0000100??????????00000???0101011;
   //localparam [31:0] MLD_D              = 32'b0000100??????????00011???0101011;
   // localparam [31:0] MLD_H              = 32'b0000100??????????00001???0101011;
-  localparam [31:0] MLD_W              = 32'b0000000??????????00010???0101011;
+  localparam [31:0] MLD_W              = 32'b0000010??????????00010???0101011;
   //localparam [31:0] MMOV_MM            = 32'b00000000000??????000000010101011;
   //localparam [31:0] MMOV_MV_I          = 32'b00000100000??????00000???0101011;
   //localparam [31:0] MMOV_MV_X          = 32'b00000010000??????00000???0101011;
